Makefile

```diff
@@ -195,6 +195,13 @@ bundle-update: ## Update containerImage, createdAt, and icon fields in the bundl
 bundle-reset-date: ## Reset bundle's createdAt
 	sed -r -i "s|createdAt: .*|createdAt: \"\"|;" ./bundle/manifests/$(OPERATOR_NAME)-operator.clusterserviceversion.yaml
 
+.PHONY: bundle-community
+bundle-community: bundle-k8s ## Update displayName, and description fields in the bundle's CSV
+	sed -r -i "s|displayName: Node Maintenance Operator|displayName: Node Maintenance Operator - Community Edition |;" ./bundle/manifests/$(OPERATOR_NAME)-operator.clusterserviceversion.yaml
```

Comment:
Do we want this? Operatorhub shows the CatalogSource, and we have different icon colors. Isn't that enough? 🤔

Reply:
oho, by CatalogSource you probably mean the CatalogSource from OpereatorHub when a use wants to install the operator

Reply:
yes, more precisely the "Community" badge in the UI

Reply:
oh... I successfully deleted that bug from my memory.
But, whatever we do, we should coordinate. All operators should do this then, or none. IMHO

Reply:
> yes, more precisely the "Community" badge in the UI

It is noticeable but not the best UX according to the options they (OLM) have suggested to us in the past emails.

Reply:
> and ATM [SNR](https://operatorhub.io/operator/self-node-remediation), [NMO](https://operatorhub.io/operator/node-maintenance-operator) and [NHC](https://operatorhub.io/operator/node-healthcheck-operator) follow this suffix

ooohhhh.... I missed that, sorry!